README.md

```diff
@@ -71,7 +74,7 @@ This code would make most sense as a...
 - ...helper method.
 
 ```text
-Your answer...
+I would code as a model method because it dealing with song model . you would use controller methods only for logic not function you want to keep controller skinny.
```

Comment:
We're actually looking for a *helper* method. These are meant just to make HTML or text show up in a view, like `form_for` and `link_to`. Model methods are used to do stuff with data that does not necessarily show up in the view. But you argued your point well, so I'll give you a :white_check_mark: !